publish-release.sh

```diff
@@ -32,13 +35,17 @@ increment_version() {
 
 # Get new version
 new_version=$(increment_version "$latest_tag" "$commit_message")
-cmd_args=""
+prerelease_flag=""
 
 if [[ $CIRCLE_BRANCH != "master" ]]; then
-  new_version="$new_version-$CIRCLE_SHA1"
-  cmd_args="--draft"
+  new_version="${new_version}-${CIRCLE_SHA1}-beta"
+  prerelease_flag="--prerelease"
 fi
 
-release_url=$(gh release create "$new_version" ./providers/*.jar --title="$new_version" --notes="$commit_message" $cmd_args)
+echo "New version: $new_version"
+echo "Commit message: $commit_message"
+echo "Prerelease flag: $prerelease_flag"
 
-echo $release_url
+release_url=$(gh release create "$new_version" ./providers/*.jar --title="$new_version" --notes="$commit_message" $prerelease_flag)
```

Comment:
nit: If that fails like last time, could we check the output and if it's not successful, we return a non success code? That way circleci will notify us

Reply:
Updated the PR